File: backend/routes/barberRoutes.js
// cortate/backend/routes/barberRoutes.js

const express = require('express');
const router = express.Router();
const auth = require('../middleware/authMiddleware');
const {
	crearActualizarPerfil,
	obtenerBarberos,
	obtenerBarberoPorId,
	toggleDisponibleAhora,
	obtenerEstadisticas
} = require('../controllers/barberController');

// Crear o actualizar perfil de barbero
router.post('/perfil', auth, crearActualizarPerfil);

// Obtener todos los barberos (público)
router.get('/', obtenerBarberos);

// Obtener un barbero específico
router.get('/:id', obtenerBarberoPorId);

// Cambiar disponibilidad (tipo Uber)
router.patch('/toggle-disponibilidad', auth, toggleDisponibleAhora);

// Obtener estadísticas (dashboard)
router.get('/dashboard/stats', auth, obtenerEstadisticas);

module.exports = router;

File: backend/routes/barberRoutes.py
# cortate/backend/routes/barberRoutes.py

from flask import Blueprint
from controllers.barberController import barber_controller

# Registrar blueprint para rutas de barberos
barber_bp = Blueprint('barber_bp', __name__)

# Enlazar rutas del controller
barber_bp.register_blueprint(barber_controller, url_prefix="/")
